pkg/multicluster/razee/razee_cluster_manager.go

```diff
@@ -361,6 +362,32 @@ func NewRazeeManager(url string, login string, password string, clusterGroup str
 	}, nil
 }
 
+func NewRazeeOAuthManager(url string, apiKey string, clusterGroup string) (multicluster.ClusterManager, error) {
+	auth, err := apikey.NewClient(apiKey)
+	if err != nil {
+		return nil, err
+	}
+	con, _ := client.New(url, http.DefaultClient, auth)
+	logger := ctrl.Log.WithName("ClusterManager")
```

Comment:
In `NewSatConfManager` you use `ctrl.Log.WithName("RazeeManager")`. 

I suggest to either use "RazeeManager" here and in local too, or give unique names if it makes sense and could help with debugging.